### ISSUE TITLE
Seems to be obsolette

### ISSUE THREAD

**Issue by s1ddok** [troughton/SwiftBGFX#1]
Sorry for raising an issue, didn't find any other way to contact. 

I found your repositories while searching for own, and wanted to let you know that we also working for SwiftPM complying. I personally did reogranization of SwiftBGFX folder structure, so it has Package.swift in the root. I'm also thinking about creating separate repository for Cbgfx.

Consider contributing to original repo, in case interested. 

Cheers.

**Comment by troughton** [comment]
Hi s1ddok,

Thanks for the heads up. This repository is more for use in a particular project and is intended to mainly track the main repository – the few changes I've made here (apart from the layout) are changing the dependencies to point to other versions of BGFX (our Cbgfx mirror isn't completely aligned with the base repo).

If we make any substantial changes, we'll definitely consider submitting pull requests to the main repository, but at this stage the only thing remotely worth submitting is the non-macOS Timing.swift implementation, and I'm not sure that's correct/best practice yet.

**Comment by s1ddok** [comment]
Yeah, I appreciate your non-macOS effort, I temporary copied your implementation. SwiftBGFX at the heart of [our engine](https://github.com/s1ddok/Fiber2D) and we are now porting to linux as well.
